src/translations/en.json

```diff
@@ -53,6 +53,7 @@
     "add_address_contacts_book": "Add address to your contacts book",
     "enter_amount": "Enter an amount",
     "advanced_settings": "Advanced settings",
+    "fee_options": "Fee Options",
```

Comment:
For the English intl, maybe we could simplify this and just write "Fee" instead of "Fee options"? 

What do you think?